### PR TITLE
Qr tab ui enhancement

### LINKED ISSUE #680: QRTab nitpicking #2 - QrCode in vertically centered etc ...
UI-part of #660

- i would add some padding between the hint "Hold your camera over the QR code" and the lower screen border

- can we have the qr code and the hint on the show-section centered vertically, if room? if there is no room, of course, scrolling is required, esp. as the hint may get longer, depending on the locale. @nayooti iirc you figured out a similar smart solution for the welcome sceen :)

### PR DESCRIPTION
**Pull request by nayooti**
fixes #680 

This is the UI-part of #660.

UI elements are vertically centered. On small devices the content is scrollable (can be triggered on SE landscape). Very same principle as WelcomeViewController.

### CODE REVIEW

File: deltachat-ios/Controller/QrPageController.swift [hunk 1]
```diff
@@ -33,6 +33,12 @@ class QrPageController: UIPageViewController, ProgressAlertHandler {
     // MARK: - lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        // temp fix - ToDo: check if this is needed for >iOS13 devices
+        if #available(iOS 13, *) {}
```
Comment (cyBerta) [inline deltachat-ios/Controller/QrPageController.swift:38]: lint is complaining here. can you put the else into the same line?

File: deltachat-ios/Controller/QrViewController.swift [hunk 2]
```diff
@@ -23,93 +36,144 @@ class QrViewController: UITableViewController {
...
         label.numberOfLines = 0
         label.textAlignment = .center
         label.font = UIFont.systemFont(ofSize: 14)
         return label
+    }()
+
+    private let container = UIView()
+
+    var minContainerHeight: CGFloat = 0 {
+        didSet {
```
Comment (cyBerta) [inline deltachat-ios/Controller/QrViewController.swift:110]: cool, I didn't know that this is possible

**Comment by cyBerta** [comment]
I think this part 

> although i stated things differently in the past, i think it is worth remembering the selected section (show or scan) between program starts, so it would be great if it can be persisted in addition to the main selection (qr, chats, settings)

is not implemented yet. Probably on purpose to have a fallback for not working cameras?

**Comment by nayooti** [comment]
> I think this part
> 
> > although i stated things differently in the past, i think it is worth remembering the selected section (show or scan) between program starts, so it would be great if it can be persisted in addition to the main selection (qr, chats, settings)
> 
> is not implemented yet. Probably on purpose to have a fallback for not working cameras?

As said. I split up #660 into #680 + #679  because somehow I had two failed attempts in implementing #679. I will have another try in #679 after #696 has been merged. I also will take care on #688 later.

File: deltachat-ios/Controller/QrPageController.swift (hunk 1 above)
Comment (nayooti) [reply deltachat-ios/Controller/QrPageController.swift]: The code block became obsolete with lastest master

**Comment by nayooti** [comment]
I removed a temp fix, rebased and forced push.

**Comment by cyBerta** [comment]
lgtm :) yeah

**Pull request merged**